### PR TITLE
Release 2.7.0 doc fixes

### PR DESCRIPTION
**Pull request by awholegunch**
[Fix Security document formatting](https://issues.cask.co/browse/CDAP-1105)
- All the tables now line up correctly.
- Shown at http://stg-web101.sw.joyent.continuuity.net/cdap/2.7.0-SNAPSHOT-2.7.0_doc_fixes/en/admin-manual/installation/security.html

[Change Search Engine for Documentation](https://issues.cask.co/browse/CDAP-1039)
- New search engine (Google searching) can be seen at:
http://stg-web101.sw.joyent.continuuity.net/cdap/2.7.0-SNAPSHOT-2.7.0_doc_fixes/en/search.html

- There is also a "Search" menu item for that page in the left sidebar, and mention on the index page.

Release Notes updated for Security change
- http://stg-web101.sw.joyent.continuuity.net/cdap/2.7.0-SNAPSHOT-2.7.0_doc_fixes/en/reference-manual/release-notes.html#release-2-7-0

### CODE REVIEW

**Comment by gokulavasan** [comment]
When I searched for "services" and clicked the Search this release (google), it returned no results :

Your search - services site:docs.cask.co/cdap/2.7.0%20OR%20docs.cask.co/cdap/2.7.0-SNAPSHOT - did not match any documents.

**Comment by awholegunch** [comment]
Looks like there are a number of problems:
- Google hasn't indexed ``docs.cask.co/cdap/2.7.0``, so nothing will get returned. That should go away, —and maybe I should add to the release practice asking Google to index the site—but:
- the syntax is wrong; "OR" doesn't work for ``site:``, it should just be `` site:docs.cask.co/cdap/2.7.0``, and ``-SNAPSHOT`` will be included automatically. (Confirmed by searching with ``2.5``)
- adding ``intext:`` to the start would mean links and URLs would get ignored, which might be a good thing. If it's not in the text, people probably don't want it. Yet, then the number of hits goes from the hundreds to the thousands. Need to think about that one.

File: cdap-docs/_common/_source/index.rst [hunk 1]
```diff
@@ -106,6 +106,12 @@ on the installation, monitoring and diagnosing fully distributed CDAP in a Hadoo
 - |faq|_ Answers to questions you might have about CDAP
 
 
+.. |search| replace:: **Search:**
+.. _search: search.html
+
+- |search|_ Search this documentation, using either *Quick Search* or *Google*
```
Comment (jwang47) [inline cdap-docs/_common/_source/index.rst]: So now we have a widget for Google searching on cdap-docs?

**Comment by jwang47** [comment]
Just one question.

File: cdap-docs/_common/_source/index.rst (hunk 1 above)
Comment (jwang47) [inline cdap-docs/_common/_source/index.rst]: Ah, I just saw the staged docs. I think it's kind of messy UI/UX, but probably fine for now.

**Comment by jwang47** [comment]
LGTM

File: cdap-docs/_common/_source/index.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/_common/_source/index.rst]: No, we have both. Google search does the site-wide; Quick Search restricts itself to the doc set, but is not as extensive. Both are implemented with a simple HTML form, such as
````
.. raw:: html

  <form action="http://www.google.com/search" name="searchbox" method="get" /> 
    <input type="hidden" name="hl" value="en" /> 
    <input type="hidden" name="ie" value="ISO-8859-1" /> 
    <input type="hidden" name="sitesearch" value="docs.cask.co/cdap/" /> 
    <input maxlength="256" size="40" name="q" value="" /> 
    <input type="submit" value="Search using Google" name="btnG" 
      style="font-size:75%;" /> 
  </form>
````

**Comment by awholegunch** [comment]
OK, I still need to fix the search though...

File: cdap-docs/_common/_source/index.rst (hunk 1 above)
Comment (jwang47) [inline cdap-docs/_common/_source/index.rst]: I see. So both Quick Search and Search with Google are using Google Search? Even after your explanation though, I'm still confused. Just by looking at the page, I wouldn't know what button to use to search.

**Comment by sreevatsanraman** [comment]
Spoke to JJ, we will merge it now and fix the google search against release.

**Pull request merged**